django/apps/aggregated/management/commands/update_aggregated_data.py

```diff
@@ -45,19 +47,20 @@
         -- Aggregate data by group
         user_data as (
             SELECT
-              R.project_id,
-              R.group_id,
-              R.user_id,
-              MAX(R.timestamp::date) as timestamp_date,
-              MIN(R.start_time) as start_time,
-              MAX(R.end_time) as end_time,
-              COUNT(DISTINCT R.task_id) as task_count,
+              MS.project_id,
+              MS.group_id,
+              MS.user_id,
+              MAX(MS.start_time::date) as timestamp_date,
+              MIN(MS.start_time) as start_time,
+              MAX(MS.end_time) as end_time,
+              COUNT(DISTINCT MSR.task_id) as task_count,
```

Comment:
you can get this value from  `mapping_sessions.items_count`

Reply:
Sure, after confirmation in Related to https://github.com/mapswipe/python-mapswipe-workers/pull/607/files/f7ef317bb36b2736dfb7d4471acb9606831bde18#diff-cb03d32601d78f2f4e96b346f6c0dcae523d8301adb4a1e4ae35838a73def35a I will make the change.

Reply:
I have pushed recommended changes.